### ISSUE TITLE
Add an Info Sheet/Cover Page for exports to explain colored text

### ISSUE THREAD

**Issue by DerekMolnar** [tuwien-csd/damap-backend#284]
**Description**
Colored text is unclear to users.  An info sheet or cover letter is needed to explain what the color(s) signify

**Describe the solution you'd like**
Create sheet with table explaining colors used in text

**Describe alternatives you've considered**
A table within the existing templates could also be used, but the concern is that users might easily overlook the table.  By using an entire sheet there is a higher chance of the user reading the text

**Additional context**
Feedback from Tomasz' DMP class.

**Comment by ChristianeStork** [comment]
Draft of the cover sheet text for the FWF template:
![image](https://github.com/user-attachments/assets/e15e0eb3-8f2c-4d88-ad7c-71fc1aafd9de)
 @ValentinFutterer, please check if the examples given are still valid.

**Comment by ValentinFutterer** [comment]
General draft looks good, it covers everything important.
**DELETE II** can be removed, since the table is being already deleted and we can do the same for the green text.

**Comment by ChristianeStork** [comment]
Thanks. And what about deleting the FWF guide (part 2 in Delete III), is this still an open issue in DAMAP?

Are you planning to add such an information page to the DAMAP FWF template soon, or should we do this in the TU Wien DMP Tool template?

**Comment by ValentinFutterer** [comment]
Good point: We should be able to do it this sprint, if you want. Then it will be in the next DMP Tool release.

**Comment by ChristianeStork** [comment]
Here are my updated ideas for the cover pages:

![Image](https://github.com/user-attachments/assets/7e002245-1c49-4e0d-b15c-895e69788842)

**Comment by ChristianeStork** [comment]
![Image](https://github.com/user-attachments/assets/b041c18d-3731-4093-8afe-5d34e7136c9f)

**Comment by ChristianeStork** [comment]
![Image](https://github.com/user-attachments/assets/f7527e25-4f3e-4c02-a93f-fc755535e353)

**Comment by ChristianeStork** [comment]
[DMP_infotexte_20250225.docx](https://github.com/user-attachments/files/18959739/DMP_infotexte_20250225.docx)
